tools/report.collector

```diff
@@ -636,9 +661,23 @@ get_logfiles() {
                     logfile=$(getcfvar $cf_type logfile "$cf_file")
                 fi
             fi
+            cs_facility=$(getcfvar $cf_type logfacility "$cf_file")
+            [ "" = "$cs_facility" ] && cs_facility="daemon"
+            find_syslog "$facility_cs.info"
 	    ;;
     esac
 
+#    . @CONFIGDIR@/pacemaker
+#
+#    facility="$PCMK_logfacility"
+#    if [ -z "$facility" ]; then
+        facility="daemon"
+#    fi
+
```

Comment:
I think that reason of **comment out** is because ```@CONFIGDIR@``` is not converted,
but for us it is important to read from config.
Could you please merge these?
 * https://github.com/inouekazu/pacemaker/commit/f23a2c6959ee83269b0ed6d89abba5e60636391c
 * https://github.com/inouekazu/pacemaker/commit/d7eb40b7146b58e5e82c478b7ec84c7a81c36b28

Should I send a pull request to 1.1 branch?

Reply:
Yes, thank you.

The new commits are fine as they are, but FYI, we do not really need to check for @CRM_LOG_DIR@/pacemaker.log because Pacemaker 1 hardcodes /var/log/pacemaker.log as the default.